src/fns.c

```diff
@@ -43,6 +43,7 @@ along with GNU Emacs.  If not, see <https://www.gnu.org/licenses/>.  */
 
 enum equal_kind { EQUAL_NO_QUIT, EQUAL_PLAIN, EQUAL_INCLUDING_PROPERTIES };
 bool internal_equal (Lisp_Object, Lisp_Object, enum equal_kind, int, Lisp_Object);
+void hash_clear (struct Lisp_Hash_Table *h);
```

Comment:
maybe a comment here saying this has moved to Rust?

Reply:
It hasn't yet moved to Rust. We need to add this declaration because this function was previously static, but since I am now calling it from Rust, it cannot be marked static.

Reply:
Ah. Carry :-)